src/homework/day8/collections/Butterflies.java

```diff
@@ -0,0 +1,52 @@
+package homework.day8.collections;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+//Создать пустой список стрингов butterflies (через new ArrayList)
+//Добавить в список 4 бабочек (Common blue, Swallowtail, Aglais io, Common blue)
+//Проитерировать список через for-each и отпечатать слова в консоль в кавычках
+//Посчитать сколько бабочек содержат букву "о" и вывести число в консоль
+//Проитерировать список по индексу и отпечатать слова в консоль через пробел
+//Проитерировать список через for-each и отпечатать слова в консоль через перенос строки
+
+public class Butterflies {
+    public static void main(String[] args) {
+
+
+        List<String> butterflies = new ArrayList<>();
+        butterflies.add("Common blue");
+        butterflies.add("Swallowtai");
+        butterflies.add("Aglais io");
+        butterflies.add("Common blue");
+
+        for (String butterfly : butterflies) {
+            System.out.println("\"" + butterfly + "\"");
+        }
+
+        int count = 0;
+        for (String butterfly : butterflies) {
+            if (butterfly.contains("o")) {
```

Comment:
nice to add kind of "toLowerCase()" to avoid situations when "O" in upperCase missed

Reply:
Fixed.